src/activities/morse_code/FirstScreen.qml

```diff
@@ -0,0 +1,125 @@
+/* GCompris - FirstScreen.qml
+ *
+ * Copyright (C) 2016 Souradeep Barua <sourad97@gmail.com>
+ *
+ * Authors:
+ *   Souradeep Barua <sourad97@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+import GCompris 1.0
+import "../../core"
+
+
+Image {
+    id: first_screen
+    anchors.fill: parent
+    fillMode: Image.PreserveAspectCrop
+    source: "qrc:/gcompris/src/activities/morse_code/resource/intro_bg.svg"
+    sourceSize.width: Math.max(parent.width, parent.height)
+    visible:false
+
+    GCText {
+        id: heading
+        text: qsTr("Exploring Morse Code")
+        fontSize: largeSize
+        horizontalAlignment: Text.AlignHCenter
+        font.weight: Font.DemiBold
+        anchors.centerIn: parent.Center
+        color: "#2a2a2a"
+        width: parent.width
+        wrapMode: Text.WordWrap
+    }
+
+    Image {
+        id: introChar
+        source: "qrc:/gcompris/src/activities/morse_code/resource/Samuel_Morse.jpg.svg"
+        sourceSize.width: parent.width * 0.25
+        fillMode: Image.PreserveAspectFit
+        verticalAlignment: Image.AlignTop
+        anchors {
+            top: heading.bottom
+            topMargin: 30 * ApplicationInfo.ratio
+            bottom: parent.bottom
+            bottomMargin: 60 * ApplicationInfo.ratio
+            left: parent.left
+            leftMargin:  30 * ApplicationInfo.ratio
+        }
+    }
+
+    GCText {
```

Comment:
I was more thinking of the window where all the characters are displayed (BrailleMap).
This one is good also except that, on phone, I don't see the whole text and tux is really too small and I can't click on it.